NightscoutServiceKit/RemoteCommands/V2/SULoopBoolSetting.swift

```diff
@@ -0,0 +1,53 @@
+//
```

Comment:
REMOTE 2.0: WILL REMOVE PRE-MERGE

Actually this is a hack I just use personally. I'll remove this before we merge.